Lib/test/test_pickle.py

```diff
@@ -397,13 +397,18 @@ def test_import(self):
         modules |= {module for module, name in REVERSE_NAME_MAPPING}
         modules |= {module for module, name in NAME_MAPPING.values()}
         for module in modules:
+            if module == "tkinter.tix":
+                # Fails when python is run with the -We flag (bpo-44785)
+                continue
```

Comment:
Since it is repeated for every use of `getmodule()`, would it be worth to move it into `getmodule()`?

Reply:
I'm sightly tentative since we'd need to return e.g. ``None``, and hence a caller that actually wants the module would have to check the return before using it. For this reason, #27538 may be a better solution.